modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/beanValidation.mustache

```diff
@@ -0,0 +1,53 @@
+{{#required}}
```

Comment:
This, `beanValidationPathParams.mustache`, and `beanValidationQueryParams.mustache` are *very* similar.  I understand the arguments for avoiding reuse across language generators, but I simply don't agree that needs to be the case within a language generator.  I think these three files need refactoring to remove the duplication.